themes/hugo-tutorials/layouts/partials/page_cta.html

```diff
@@ -1,6 +1,6 @@
 <div class="page-cta">
 	<ul>
-		<li class="order-1 order-lg-0 d-none d-lg-flex other-menu"><a href="https://slash.dgraph.io" title="Slash" target="_blank" class="button button--primary button--nav justify-content-center justify-content-xl-start download-btn"><span>Get Slash GraphQL</span><i class="fa fa-arrow-right d-none d-xl-block" aria-hidden="true"></i></a></li>
+		<li class="order-1 order-lg-0 d-none d-lg-flex other-menu"><a href="https://slash.dgraph.io" title="Dgraph Cloud" target="_blank" class="button button--primary button--nav justify-content-center justify-content-xl-start download-btn"><span>Get Dgraph Cloud</span><i class="fa fa-arrow-right d-none d-xl-block" aria-hidden="true"></i></a></li>
```

Comment:
The link here is also now cloud.dgraph.io FYI

Reply:
Fixed, thanks!